doc/man/man3/crypto_verify16.3monocypher

```diff
@@ -88,7 +88,7 @@ network along with a message, but the correct MAC is secret.
 If the attacker attempts a forgery, one does not want to reveal
 .Dq your MAC is wrong, Em and it took 384 microseconds to tell .
 If the next attempt takes 462 microseconds instead, it tells the
-attacker they just guessed a few bytes correctly.
+attacker they just guessed a byte correctly.
 That way, an attacker can derive the correct MAC, and successfully
```

Comment:
Good. But now I'm tempted to modifiy line 92 thus:

    That way, an attacker can derive the correct MAC _byte by byte_, [...]

(Emphasis just shows the diff, don't use `.Em`)